dev/io.openliberty.jacc.2.0/bnd.bnd

```diff
@@ -0,0 +1,27 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion: 1.0
+
+Bundle-Name: JACC 2.0 Service Provider Interface
+Bundle-Description: JACC 2.0 Service Provider Interface, version ${bVersion}
+Bundle-SymbolicName: io.openliberty.jacc.2.0
+
+Import-Package: com.ibm.wsspi.security.authorization.jacc
+
+Export-Package: com.ibm.wsspi.security.authorization.jacc
+
+-includeresource: {META-INF/maven/io.openliberty.api/io.openliberty.jacc.2.0/pom.xml=io.openliberty.jacc.2.0.pom}
+
+publish.wlp.jar.suffix: dev/api/ibm
+
+-buildpath: \
+	com.ibm.ws.security.authorization.jacc
```

Comment:
Shouldn't this be io.openliberty.security.authorization.internal.jacc?

Reply:
After talking this over with @jhanders34 and @aguibert , we decided to remove the `io.openliberty.jacc.2.0` project and `jakartaeeMe` the `com.ibm.websphere.appserver.api.jacc` project instead.

The reason being that currently the jakartaeeMe'd bundles are not published to the local BND repo. Even if they were, that would not resolve issues it would present in Eclipse. This still leaves the issue of the POM files not being updated and renamed but there are several other projects with the same issue that will need to have the same issue resolved.